addons/medical_engine/functions/fnc_getHitpointArmor.sqf

```diff
@@ -0,0 +1,69 @@
+#include "script_component.hpp"
+/*
+ * Author: Pterolatypus
+ * Checks a unit's equipment to calculate the total armor on a hitpoint
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Hitpoint <STRING>
+ *
+ * Return Value:
+ * Total armor for the given hitpoint <ARRAY>
+ * 0: Armor <NUMBER>
+ * 1: ExplosionShielding <NUMBER>
+ *
+ * Example:
+ * [player, "HitChest"] call ace_medical_engine_fnc_getHitpointArmor
+ *
+ * Public: No
+ */
+params ["_unit", "_hitpoint"];
+
+// Helper function to check cache and/or read config
+private _fnc_getItemArmor = {
+    params ["_item", "_hitpoint"];
+
+    private _key = format ["%1$%2", _item, _hitpoint];
+    private _armor = GVAR(armorCache) getVariable _key;
+
+    if (isNil "_armor") then {
+
+        TRACE_2("Cache miss", _item, _hitpoint);
+        if (_item isEqualTo "" || _hitpoint isEqualTo "") exitWith {
```

Comment:
This doesn't really matter as it is cached, but you could also use something like:
```sqf
"" in [_item, _hitpoint]
```